pubsub/subscriptions/main.go

```diff
@@ -167,6 +170,37 @@ func pullMsgsSettings(client *pubsub.Client, subName string) error {
 	return nil
 }
 
+func pullMsgsConcurrencyControl(client *pubsub.Client, subName string, numGoroutines int) error {
+	ctx := context.Background()
+	// [START pubsub_subscriber_concurrency_control]
+	sub := client.Subscription(subName)
+	sub.ReceiveSettings.NumGoroutines = numGoroutines
```

Comment:
I would consider adding a comment here just to clarify that `numGoroutines` threads are used for  executing callbacks.